src/shared/components/header/Header.tsx

```diff
@@ -10,9 +17,21 @@ export const Header: React.FC<Props> = ({ onLogin }) => {
     <header css={wrapper}>
       <p>Router Exercise</p>
       <div css={linkWrapper}>
-        <a css={link}>Home</a>
-        <a css={link}>Info</a>
-        <a css={link}>Blog</a>
+        <NavLink to="/" css={link}>
+          {({ isActive }) => (
+            <p css={isActive ? activeLink : undefined}>Home</p>
+          )}
```

Comment:
Ne potrebno je stavljati p unutar NavLink komponente, mozes maknuti <p> i prebaciti css logiku direktno u NavLink. Primjer iz dokumentacije:

` <NavLink
            to="tasks"
            className={({ isActive }) =>
              isActive ? activeClassName : undefined
            }
          >
            Tasks
          </NavLink>`

Reply:
https://reactrouter.com/docs/en/v6/components/nav-link